k8s/migratev2/templates/job.yaml

```diff
@@ -78,7 +78,6 @@ spec:
             name: config2
           - mountPath: {{ dir .Values.credentials }}
             name: creds-secret
-            key: {{ base .Values.credentials }}
```

Comment:
this provides the key in the secret file... How can you get rid of this?
(hint: if you want to use the service account, then there are no credentials  at all, but we need a service account that knows how to  deal with gcs buckets)

Reply:
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Job.spec.template.spec.containers[0].volumeMounts[3]): unknown field "key" in io.k8s.api.core.v1.VolumeMount
```
Without it, k8s mounts all keys in the secret. I think the key param is for Volumes and not VolumeMounts.